lib/event-bus.js

```diff
@@ -10,22 +10,21 @@ module.exports = class EventBus {
     log.info('EventBus initialized');
   }
 
-  watch (DOMStorage) {
-    DOMStorage.domStorageItemUpdated(({ storageId, key, newValue }) => {
-      if (key !== 'bus') {
-        return;
-      }
+  handleDOMStorageChange ({ key, newValue }) {
+    if (key !== 'mocha-chrome-bus') {
+      return;
+    }
 
-      if (newValue === '') {
```

Comment:
we still need this check. there was some error with JSON.parse and empty values, which did happen.

Reply:
It happened because you were initializing it with `''` in [client/event-bus.js here](https://github.com/shellscape/mocha-chrome/pull/17/files#diff-60b148cadaeef8d6edf6024300bcb7ecL6) which has been removed, so it's not needed anymore.

Reply:
granted, but I think there was another condition that was bombing on that. or maybe I'm losing my marbles. either way, we should at least try/catch around that `JSON.parse` just in case of poop

Reply:
that makes more sense. Although I hope no one is setting `mocha-chrome-bus`, and it could end up hiding a bug, so I'd lean to leave it.